components/sections/NavItem/index.module.css

```diff
@@ -0,0 +1,39 @@
+.navItem {
+  @apply inline-flex items-center gap-2 rounded px-3 py-2;
```

Comment:
Remember to reread the [contribution guideline](https://github.com/nodejs/nodejs.org/blob/main/COLLABORATOR_GUIDE.md#creating-react-components) 😄
[Example of how it's should write](https://github.com/nodejs/nodejs.org/blob/main/components/Common/Banner/index.module.css)

Reply:
Okay Sir 😅